Logger/FileLogger.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string? _LogPath;
+        public string LogPath
+        {
+            get { return _LogPath!; }
+            set { _LogPath = value ?? throw new ArgumentNullException(); }
+        }
+        
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            StreamWriter logStreamWriter = File.AppendText(LogPath);
+            string infoLine = $"{DateTime.Now} {ClassName} {logLevel}:";
+
+            logStreamWriter.WriteLine(infoLine);
+            logStreamWriter.WriteLine(message);
+        }
+
+        public FileLogger(string logPath) => LogPath = logPath;
```

Comment:
```suggestion
        public FileLogger(string? logPath) => LogPath = logPath ?? throw new ArgumentNullException(nameof(logPath));
```
Since you don't need the public `LogPath` property, you can set your `logPath` parameter to be nullable and check for null in your constructor and throw accordingly. This will prevent you from having to use the `!` in your unit tests.